DemoApp/DemoApp/Constant/ImageConstants.cs

```diff
@@ -13,5 +13,9 @@ internal static class ImageConstants
         public const string PasswordIcon = "password_eye.png";
```

Comment:
Update closes #ticketNumber in the PR

Reply:
I added #184 in the description. Is that the requirement?

Reply:
Yeah, We should have a 'ShouldCloseWhenTappedOutside' property.

Reply:
There is already a property `CloseWhenBackgroundIsClicked` in the PopupPage.cs (used in `SampleCustomPopup.xaml` line 9).

For TMModal, The TMModal.Show() method takes that as an argument, with default value false.

Reply:
Then, it is fine.